vic/drivers/shared_image/src/vic_init.c

```diff
@@ -1219,22 +1226,12 @@ vic_init(void)
             }
         }
 
-        // Bare soil is now read in as the "last" (highest index) vegetation
-        // class
-        // rescale vegetation classes to 1.0 if their sum is greater than 0.99
-        // otherwise throw an error
-        // TBD: Need better check for equal to 1.
-        if (veg_con[i][0].Cv_sum != 1.) {
+        // Bare soil is now read in as the "last" (highest index) vegetation class
```

Comment:
The original comment was not written by me so I'll defer to @bartnijssen to clarify if necessary.  My understanding is that the order of veg classes in the veg library can be specified by the user, however, internally we store the bare soil as the last slot in the `veg_con` array.